src/styles.scss

```diff
@@ -2,6 +2,19 @@
   padding: 5px;
 }
 
+.react-kanban-form__button{
```

Comment:
I don't know what to think about the BEM name for "functions", at first I thought you've created a new element in jsx. What if we hoist this and change it for a name without BEM? Or maybe those components need to be extracted?!